AUSTA-295: Added blueprint "Install wimlib-utils on Linux CentOS 8 Stream"

h4xhor opened the pull request:
Added blueprint `Install wimlib-utils on Linux CentOS 8 Stream`.

![image](https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend/assets/14309370/78d6919e-8a72-49e5-9b7a-0fbb8f934eff)

Commit:
AUSTA-289: Added blueprint "Install Prerequisites on Windows Worker" to install 7z.exe and modified step "Extract Windows ISO on Windows Worker" to use 7z.exe.

Commit:
AUSTA-289: 
1. Modified blueprint "Install Prerequisites on Windows Worker" so the Windows PATH environment variable is no longer modified.
2. Updated step "`Extract Windows ISO on Windows Worker" `to call 7z.exe using the absolute path.

h4xhor committed:
Merge branch 'Attune-Automation:master' into master

jarrodchesney commented:
Hi @h4xhor Please remove the EPEL enable step and remove Centos 8 stream from
the name.

Please to this instead

Have an IF statement that checks if the thing you want already exists, use "which command" Don't check for a package. This allow people to preinstall dependencies if we don't yet support installing them for their OS.

Have an IF statement that checks the OS type.
If it's not RHEL, or a RHEL clone, etc, then error out with "implementation needed for this flavor of Linux"

If it is one of the RHEL or RHEL clone OSs, the check if EPEL is available or error out, then check if the package is available or error out, then install the packages.

The reason being, we can handle the package installations on different OSs pretty easily, otherwise we will have A HEAP of blueprints

Commit:
Merged pulled changes

Commit:
AUSTA-295: Added blueprint "Install wimlib-utils on Linux CentOS 8 Stream".

h4xhor commented:
Hi @jarrodchesney , I've:

1. Added checks for if `wimlib-utils` is installed using `which mkwinpeimg`. 
2. Added error checking for Linux OS type.
3. Error out if EPEL is not enabled.

![image](https://github.com/Attune-Automation/Automate-Windows-Installation-with-autounattend/assets/14309370/60f5b227-b968-42a0-89c5-c9a4f83cb085)

Commit:
AUSTA-295: 
1. Added checks for wimlib-utils installed using which.
2. Added error checking for Linux OS type.
3. Error out if EPEL is not enabled.

jarrodchesney commented:
Closed, I'll create the steps with the changes I'd like made.
